tests/test_pnytter_profiles.py

```diff
@@ -94,6 +94,35 @@
         ),
         id="@PossumEveryHour",
     ),
+    pytest.param(
+        "nobio",
+        TwitterProfile(
+            id=14814846,
+            username="nobio",
+            fullname="Gernot",
+            biography=None,
```

Comment:
The Pydantic model for TwitterProfile declares `biography` as a string, so it must be set as empty string (which is what we're returning from the parser anyway). However I don't know wether it may be better to return None or empty string when a user has no bio. :thinking: 

```suggestion
            biography="",
```

Reply:
I think that an empty string is better as it is easier to handle. You can just print out an empty string without handling the NoneType. I will change this.